test-worker/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "test": "adobe-asset-compute test-worker"
+    "test": "aio info && adobe-asset-compute test-worker"
```

Comment:
keeping this so the version gets logged in dep mob tests that don't install aio in the github action workflow